streams/bluetooth.py

```diff
@@ -0,0 +1,176 @@
+#! /usr/bin/python3
+# -*- coding: utf-8 -*-
+
+# Uses rtl_fm to access an RTL-SDR dongle, redsea to process RDS data (if any), and aplay to output to audio device
```

Comment:
Probably make a comment about what this script does. The rtl_fm comment is old and should be removed.

Reply:
You may want to run all these python scripts through pylint just to pick up on some weird convention stuff.